packages/schema/src/schema/project/sources/geojsonsource.json

```diff
@@ -12,7 +12,7 @@
     "data": {
       "type": "object",
       "description": "The GeoJSON data",
-      "$ref": "./geojson.json"
```

Comment:
Should still be "./geojson.json" since it's located next to it?

Reply:
You were right using "./project/sources/geojson.json" it seems, because we use `cwd` as `./src/schema` in here https://github.com/geojupyter/jupytergis/blob/main/packages/schema/package.json#L30

Reply:
But it seems `datamodel-codegen` prefers `"$ref": "./geojson.json"`, which, to be fair, makes more sense

Reply:
Dude 😓 `"$ref": "./geojson.json"` is the correct approach IMO. `datamodel-codegen` agrees.

But json2ts does not like that https://github.com/bcherny/json-schema-to-typescript/issues/324